implementation-contributed/v8/mjsunit/harmony/to-number.js

```diff
@@ -19,6 +19,8 @@ assertEquals(NaN, %ToNumber(undefined));
 assertEquals(-1, %ToNumber("-1"));
 assertEquals(123, %ToNumber("123"));
 assertEquals(NaN, %ToNumber("random text"));
+assertEquals(NaN, %ToNumber("INFINITY"));
+assertEquals(NaN, %ToNumber("infinity"));
```

Comment:
Kudos for thoroughness, but I don't want to get in the habit of maintaining these files.

Reply:
I'm good with contributors updating these on a best-effort basis if they want to.

Reply:
Can't say I understand that, but [given the precedence for modifying these files](https://github.com/tc39/test262/commits/main/implementation-contributed), I'll take that discussion elsewhere.

Reply:
@jugglinmike this change: https://github.com/tc39/test262/commit/805e081783f43e4857839167837e1bd8e5551c46 was just an oversight. Generally though, these tests shouldn't change, except in the case of removal (ie. they've been reviewed and anything useful has been moved into `test/`